### ISSUE TITLE
Voiceover navigation is not working on web staging and autopush URLs

### ISSUE THREAD

**Issue by asarangam** [flutter/flutter#115306]
https://user-images.githubusercontent.com/118222744/201767757-e8ef7931-ddca-4717-b7b0-c31ed7062438.mp4

https://user-images.githubusercontent.com/118222744/201767877-9bc23b02-e3d5-4f1f-9a1d-e13836fd8bc5.mp4

https://user-images.githubusercontent.com/118222744/201767946-2690b325-5720-4111-915f-996af2521bc2.mp4

https://user-images.githubusercontent.com/118222744/201767991-eb7199cc-a723-46dc-8e75-4fb45fdb8a75.mp4

Account:  [uni.accessibility@gmail.com](mailto:uni.accessibility@gmail.com) 
MacOS 12.6.1, Windows
Screen reader: Voiceover, NVDA

URLs:
https://familylink-staging.corp.google.com/
https://familylink-autopush.corp.google.com/

Steps to reproduce the issue:
1. Open above staging or autopush web URL.
2. Try to navigate on the page using both tab key and also using screen reader navigation keys (control + option + left / right arrows).

Expected result:
Screen reader navigation should work on the pages using both tab key and also screen reader navigation keys (control + option + left / right arrows) to focus and activate the controls.

Actual result:
Screen reader navigation is not working on the pages using both tab key and also screen reader navigation keys (control + option + left / right arrows) to focus and activate the controls.
Only keyboard focus is moving on the screen and screen reader focus is not visible.

One time I was able to focus on the hidden "Enable accessibility" button and activate it. Then screen reader started working.
But most of the time, when navigation is started, screen reader focus is not going to the hidden enable a11y button or no info is read by screen reader about hidden enable a11y button.
One time I heard screen reader reading as "Item removed" when I tried activating hidden enable a11y button and screen reader was still not working.

Also, as we have custom enable a11y button, we should have a documentation about this feature on how to enable it as per below GAR guideline. Could you please suggest.
1.17. If you have custom or non-standard accessibility features, are they documented in your Help Center?

I am able to repro this issue when I have both autopush and staging URLs open on the browser in different tabs.

"Enable accessibility" button is associated with hamburger main menu, where on activating this button, screen reader focus is activating hamburger menu also and screen reader is reading as "Expand navigation menu".

Sometimes, on activating "Enable accessibility" button, screen reader is reading as "Item was removed".

GAR Failed:
1.16. Final Run-Through: Can the use case be completed in the following scenarios?
(c) Without Visible Screen (With Screen Reader)

https://user-images.githubusercontent.com/118222744/201767606-f3b7181f-fec2-486a-840f-6dba27593805.mp4

https://user-images.githubusercontent.com/118222744/201767659-4045e1bd-9a67-412f-8fbb-1c6033061ef9.mp4
